### DISCUSSION TITLE
How to turn off copilot?

### DISCUSSION THREAD

**Discussion by tumugide** [zed-industries/zed#18001] [Feature Requests]
Hello,
I tried to disabled auto completion and switched inline_completions to none but it doesn't work at all.
How do I do it properly or it just doesn't work?

```json
{
  "show_inline_completions": false,
  "assistant": {
    "default_model": {
      "provider": "zed.dev",
      "model": "claude-3-5-sonnet-20240620"
    },
    "version": "2"
  },
  "theme": "Andromeda",
  "format_on_save": "true",
  "ui_font_size": 14,
  "buffer_font_size": 14,
  "soft_wrap": "editor_width",
  "show_completions_on_input": false,
  "features": {
    // "inline_completion_provider": "copilot"
    "inline_completion_provider": "none"
  },
  "always_treat_brackets_as_autoclosed": true,
  "autosave": "on_focus_change",
  "ui_font_family": "FiraCode Nerd Font",
  "buffer_font_family": "FiraCode Nerd Font",
  "project_panel": {
    "auto_fold_dirs": true,
    "default_width": 300,
    "dock": "right"
  }
}```

**Comment by bajrangCoder** [comment]
For me either of the following settings disables the copilot:

   ```jsonc
   {
     "features": {
       "inline_completion_provider": "none"
     }
   }
   ```
Or
   ```jsonc
   {
     "show_inline_completions": false
   }
   ```

I’m using **Zed version 0.152.6**.
Try anyone of them or try updating the zed

**Comment by xpe** [comment]
Style tip: if you use the `jsonc` code fence (instead of `json`), the comments won't get highlighted in red. Why? [JSONC allows comments but JSON does not](https://changelog.com/news/jsonc-is-a-superset-of-json-which-supports-comments-6LwR).

**Reply by xpe** [reply]
The comment above is intended for the OP, @tumugide, in case they want to edit the top-level post (or use the tip in the future)

**Comment by bla-k** [comment]
This has been changed to
```
  "features": {
    "edit_prediction_provider": "none"
  },
```

**Reply by 0xBlitzmachine** [reply]
Thanks =}

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
